Moderation-Policy.md

```diff
@@ -79,64 +78,73 @@ Note, however, that unfamiliarity with the [Code of Conduct][] does not excuse a
 ### Guidelines and Requirements
 
 * All Posts are expected to respect the Node.js [Code of Conduct][].
-* Only Collaborators with commit rights to a given repository may Moderate Posts within that repository's issue tracker.
-* The TSC serves as the final arbiter for all Moderation issues (see: [Escalation to the TSC](#escalation-to-the-tsc)).
-* Only a TSC member may Remove or Ban an individual from the Node.js GitHub Organization.
-* For any Removal or Banning action, an issue describing the reasons for the action, and identifying the Github account being acted upon, must be posted to the Moderation Repository with an explanation provided by the TSC member performing the action.
+* Any Collaborator with commit rights to a given repository may Moderate Posts within that repository's issue tracker.
+* The Moderation Team serves as the final arbiter for all Moderation issues.
+* Moderation Team members may Remove or Ban an individual from the Node.js GitHub Organization.
+* For any Removal or Banning action, an issue describing the reasons for the action, and identifying the Github account being acted upon, must be posted to the Moderation Repository with an explanation provided by the Moderation Team member performing the action.
```

Comment:
Yeah, this is a part I'm not entirely sure on yet. org owners have access to lots of things that it may not make sense for moderators to have access to (the security repo, for instance). We need to be sure about this.